tasks/publisher.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: publisher
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Create a directory cvmfs_keys
+  ansible.builtin.file:
+    path: /tmp/keys
```

Comment:
`/var/tmp` would be better than `/tmp` since this would avoid a change after each cleanup of `/tmp`.

Reply:
done